### ISSUE TITLE
[BUG] - Can't access dashboard on unified/ban website

### ISSUE THREAD

**Issue by AlexzanDev** [unified-ban/Terminal#59]
**Describe the bug**
After accepting the login request in Telegram in the Sign In process, the unified/ban website gives me the option to enter the dashboard or sign out. However, when clicking the "Dashboard" button (which should redirect to the /Backend page), the site redirects me to the unifiedban.solutions home page and throws an error in the browser (Chrome 85.0.4183.121 on Windows 10 64-bit) related to Vue.js.

**Error**:
```
vue.runtime.esm.js:1888 Error: Redirected from "/SignIn" to "/Backend" via a navigation guard.
    at Kt (vue-router.esm.js:2060)
    at zt (vue-router.esm.js:2024)
    at vue-router.esm.js:2226
    at index.js:136
    at h (vue-router.esm.js:2212)
    at r (vue-router.esm.js:1896)
    at r (vue-router.esm.js:1900)
    at r (vue-router.esm.js:1900)
    at Dt (vue-router.esm.js:1904)
    at e.Qt.confirmTransition (vue-router.esm.js:2242)
```

**To Reproduce**
1. Go to 'Dashboard' on the header menu
2. Do the Telegram login process
3. Click "dashboard" after the login
4. See if the user gets redirected to the home page or accesses the dashboard

**Expected behavior**
I expected to access the unified/ban dashboard as usual.

**Comment by mirkobrombin** [comment]
This should be fixed in next Dashboard release (5.0) written in Blazor C#.

**Comment by mirkobrombin** [comment]
Fixed by The 2020 Winter update ❄️
